lomap/graphgen.py

```diff
@@ -106,7 +111,8 @@ def __init__(self, dbase):
         self.max_nodes = 100
 
         # The maximum threshold distance in A unit used to select if a molecule is depicted
```

Comment:
oh, yes, the max_mol_size will compared to the max_disi_mol where defined in the generate_depictions function. So the max_disi_mol return the maximum distance between any two atoms in a molecule. If the molecule is big, the max_disi_mol is large and will exceed the threshold which is defined by max_mol_size and so the molecule will be discard for depicting.